zerver/models.py

```diff
@@ -152,6 +152,7 @@ class Realm(ModelReprMixin, models.Model):
     date_created = models.DateTimeField(default=timezone.now) # type: datetime.datetime
     notifications_stream = models.ForeignKey('Stream', related_name='+', null=True, blank=True) # type: Optional[Stream]
     deactivated = models.BooleanField(default=False) # type: bool
+    default_language = models.CharField(default=u'en', max_length=50) # type: text_type
```

Comment:
So a commit  defining `MAX_LANGUAGE_ID_LENGTH` with value 50 and using this variable for the `default_language` field in UserProfile model? Should this commit be part of this pull request or should I give a separate pull request with just that commit?